### PR TITLE
Test2

### PR DESCRIPTION
**Pull request by shrutisachan08**
The pull request describes the week 2 Assignments that dealt with the implementation of topics like:
Outlier Analysis
Practical Handling of missing values
Exploratory Data Analysis
Linear Regression with and without Scikit-learn
Logistic Regression with Scikit-learn
The features and applications of machine learning were covered in an uncomplicated way.
I got better insights of cost function and gradient descent too along with linear regression model.

### CODE REVIEW

**Comment by Pin4sf** [comment]
Try to host your rating converter webpage

**Comment by shrutisachan08** [comment]
Sir can you please guide that how to host it .
